client/composables/axios/useAxiosMiddlewares.ts

```diff
@@ -42,13 +42,7 @@ export function useAxiosMiddlewares(): ComputedRef<AxiosMiddlewares> {
 
           const { setTokens } = useAuth()
           setTokens(null)
-          toast.add({
-            severity: 'error',
-            summary: 'Session',
-            detail: 'Session expired, please login again',
-            life: 3000,
-            closable: true,
-          })
+          refreshTokenExpiredEvent.emit()
           navigateTo('/login')
```

Comment:
I feel like it's no longer the responsability of the composable to redirect to login..

Reply:
fair enough